### ISSUE TITLE
Error on build

### ISSUE THREAD

**Issue by vvscode** [olignyf/raspberry-pi-showip#2]
Trying to install package and got this:

```
pi@vvscodepi:~/raspberry-pi-showip $ make
gcc -Wall `pkg-config --cflags gtk+-2.0 lxpanel` -shared -fPIC showip.c toolbox.c toolbox-char-array.c toolbox-line-parser.c toolbox-text-buffer-reader.c -o showip.so `pkg-config --libs lxpanel`
Package lxpanel was not found in the pkg-config search path.
Perhaps you should add the directory containing `lxpanel.pc'
to the PKG_CONFIG_PATH environment variable
No package 'lxpanel' found
Package lxpanel was not found in the pkg-config search path.
Perhaps you should add the directory containing `lxpanel.pc'
to the PKG_CONFIG_PATH environment variable
No package 'lxpanel' found
showip.c:30:28: fatal error: lxpanel/plugin.h: No such file or directory

                            ^
compilation terminated.
toolbox.c: In function ‘C_GetTempFilename’:
toolbox.c:636:2: warning: implicit declaration of function ‘close’ [-Wimplicit-function-declaration]

  ^~~~~
toolbox.c:640:3: warning: implicit declaration of function ‘unlink’ [-Wimplicit-function-declaration]
 
   ^     
toolbox.c: In function ‘C_System’:
toolbox.c:1404:15: warning: variable ‘result_written’ set but not used [-Wunused-but-set-variable]
 
               ^             
toolbox.c: In function ‘C_Sleep’:
toolbox.c:2092:2: warning: implicit declaration of function ‘usleep’ [-Wimplicit-function-declaration]
  char letter[3]="";
  ^~~~~~
toolbox.c: In function ‘C_getCompileDate’:
toolbox.c:2425:7: warning: variable ‘temp’ set but not used [-Wunused-but-set-variable]
  strRet = strchr(date, ' ');
       ^~~~
At top level:
toolbox.c:2100:27: warning: ‘g_rand_source_num_nozero’ defined but not used [-Wunused-const-variable=]
  {
                           ^                       
toolbox.c:2097:27: warning: ‘g_rand_source_alphanum’ defined but not used [-Wunused-const-variable=]
  }
                           ^                     
toolbox-char-array.c: In function ‘charArray_getLineThatIs’:
toolbox-char-array.c:1156:6: warning: unused variable ‘single_fail’ [-Wunused-variable]
  int single_fail = 0;
      ^~~~~~~~~~~
toolbox-char-array.c:1155:14: warning: unused variable ‘array_insider’ [-Wunused-variable]
  charArray * array_insider = NULL;
              ^~~~~~~~~~~~~
toolbox-char-array.c: In function ‘charArray_substituteInLineThatMatches’:
toolbox-char-array.c:1298:59: warning: passing argument 3 of ‘lineParser_splitTag’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
  iret = lineParser_splitTag(&parser, root->array[row][0], needle, &tempArray_ptr);
                                                           ^~~~~~
In file included from toolbox-char-array.c:28:0:
toolbox-line-parser.h:69:5: note: expected ‘char * const’ but argument is of type ‘const char *’
 int lineParser_splitTag(lineParser * parser, const char * const line, char * const tag, charArray ** out_array);
     ^~~~~~~~~~~~~~~~~~~
toolbox-char-array.c:1242:11: warning: variable ‘array’ set but not used [-Wunused-but-set-variable]
  char *** array;
           ^~~~~
toolbox-char-array.c:1238:6: warning: variable ‘index’ set but not used [-Wunused-but-set-variable]
  int index;
      ^~~~~
toolbox-char-array.c: In function ‘charArray_substituteArrayInLineThatMatches’:
toolbox-char-array.c:1418:59: warning: passing argument 3 of ‘lineParser_splitTag’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
  iret = lineParser_splitTag(&parser, root->array[row][j], needle, &tempArray_ptr);
                                                           ^~~~~~
In file included from toolbox-char-array.c:28:0:
toolbox-line-parser.h:69:5: note: expected ‘char * const’ but argument is of type ‘const char *’
 int lineParser_splitTag(lineParser * parser, const char * const line, char * const tag, charArray ** out_array);
     ^~~~~~~~~~~~~~~~~~~
toolbox-char-array.c:1370:11: warning: variable ‘array’ set but not used [-Wunused-but-set-variable]
  char *** array;
           ^~~~~
toolbox-char-array.c:1366:6: warning: variable ‘index’ set but not used [-Wunused-but-set-variable]
  int index;
      ^~~~~
toolbox-char-array.c: In function ‘charArray_to_buffer’:
toolbox-char-array.c:2699:10: warning: variable ‘buffer_malloced’ set but not used [-Wunused-but-set-variable]
  uint8_t buffer_malloced = 0;
          ^~~~~~~~~~~~~~~
Makefile:27: recipe for target 'library' failed
make: *** [library] Error 1
pi@vvscodepi:~/raspberry-pi-showip $ 
```

any way to fix?

**Comment by ThomasGugler** [comment]
same issue...project is dead?

**Comment by unlikezy** [comment]
execute `sudo apt-get install lxpanel-dev`  solve this problem.

**Comment by unlikezy** [comment]
![image](https://user-images.githubusercontent.com/3445204/113502042-35f96a00-955c-11eb-9ceb-857eccbc0fb5.png)
but won't display IP anyway…

**Comment by olignyf** [comment]
Hi, sorry for the delay.
I updated the project to work on Raspberry Pi 4, now the IP is shown for Ethernet. 
![2022-02-20-022818_1920x1080_scrot](https://user-images.githubusercontent.com/1154566/154826013-8768a655-0824-42ad-b791-bbf91d1923d5.png)
I still need to update & test for wifi.

**Comment by olignyf** [comment]
Fixed in v1.1, try again
